index.html

```diff
@@ -61,20 +62,24 @@ <h2>My List</h2>
 
         <!-- empty item -->
         <div class="empty-item" style="display:none">
-          <div class='card shadow p-4 mb-4 bg-white'>
-            <div class='card-body'>
-              <div class="float-start">
-                <div class="form-check">
-                  <input id="" class="empty-checkbox form-check-input" type="checkbox"
-                    onclick="toggleCheckboxText()"><label class="empty-label form-check-label" for=""></label>
+          <div class='card shadow p-1 mb-1 bg-white'>
+            <div class='card-body'>          
+              <div class="row">
+                <div class="col-sm-11">
```

Comment:
in a xs view port the button is still wrapping to a new line. try changing this to col-11 instead of col-sm-11